### PR TITLE
Bump Param version to 2.0.0

### PR DESCRIPTION
**Pull request by maximlt**
Ref https://github.com/conda-forge/param-feedstock/pull/25

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -20,6 +20,8 @@ requirements:
     - pip
     - setuptools
     - wheel
+    - hatchling
+    - hatch-vcs
```
Comment (maximlt) [inline recipe/meta.yaml]: - I'm not sure why `setuptools` and `wheel` are there, they are not in the conda-forge recipe.
- I've added `hatchling` and `hatch-vsc` as in the conda-forge recipe, they are indeed new build dependencies https://github.com/holoviz/param/blob/b6a285b7205a869b5677b8571fd54c2a5ccc8709/pyproject.toml#L1-L3

**Review by philippjfr — APPROVED** [review]
Looks good to me!

File: recipe/meta.yaml (hunk 1 above)
Comment (lorepirri) [reply recipe/meta.yaml]: - They switched from `setuptools` to `hatch`: https://github.com/holoviz/param/commit/aa826ef429bc13f34e0179135e160a0a022faf39
- we can remove `setuptools` and `wheel` probably

**Review by ViridianMelody — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (maximlt) [reply recipe/meta.yaml]: @lorepirri do you want me to remove `setuptools` and `wheel`?

Comment (M-Waszkiewicz-Anaconda) [reply recipe/meta.yaml]: @maximlt, yes, in the commit Lorezno mentioned, there is:
```
- python -m pip install setuptools wheel twine tox
+ python -m pip install build
```

Comment (lorepirri) [reply recipe/meta.yaml]: If also @M-Waszkiewicz-Anaconda agrees with the intuition, I would remove them. `hatch` is another build system, and I don't see `setuptools` being used anywhere. There is a reference for an optional `setuptools_scm` though.

Comment (maximlt) [reply recipe/meta.yaml]: Removed in https://github.com/AnacondaRecipes/param-feedstock/pull/7/commits/297b6053912b5ee05e5040144dd259d52fa69521.

> There is a reference for an optional setuptools_scm though.

`setuptools_scm` is used to deal with editable installs of the package, so I think it's not relevant here.

Comment (maximlt) [reply recipe/meta.yaml]: Thanks both for your answers!

**Comment by lorepirri** [comment]
@M-Waszkiewicz-Anaconda what's with the linter error?

File: recipe/meta.yaml (hunk 1 above)
Comment (lorepirri) [reply recipe/meta.yaml]: ok perfect!

**Pull request merged**